libs/utils/executor.py

```diff
@@ -464,38 +473,38 @@ def _wload_init(self, tc, wl_idx):
         wload = self._wload_conf(wl_idx, wlspec)
 
         # Keep track of platform configuration
-        self.te.test_dir = '{}/{}:{}:{}'\
-            .format(self.te.res_dir, wload.wtype, tc_idx, wl_idx)
-        os.system('mkdir -p ' + self.te.test_dir)
-        self.te.platform_dump(self.te.test_dir)
+        test_dir = '{}/{}:{}:{}'.format(self.te.res_dir, wload.wtype,
+                                        tc_idx, wl_idx)
+        os.system('mkdir -p ' + test_dir)
+        self.te.platform_dump(test_dir)
 
         # Keep track of kernel configuration and version
         config = self.target.config
-        with gzip.open(os.path.join(self.te.test_dir, 'kernel.config'), 'wb') as fh:
```

Comment:
Again, these two chunks are cosmetics... keep in the same separate patch with the previous one.